docs/installation.rst

```diff
@@ -2,7 +2,15 @@
 
 .. highlight:: none
 
-Installation instructions
+Using with Docker
+=================
+
+A docker container exists for easy access to the `tinygo` CLI.
+For example, to compile the blinky example from the root of the repository::
+
+    docker run --rm -v $(pwd):/src tinygo/tinygo build -o /src/blinky.elf -target arduino /src/examples/blinky
+
+Binary installation instructions
 =========================
```

Comment:
Nit: this underline should be the same width as the title (to avoid Sphinx warnings)

Reply:
On it

Reply:
Fixed